internal/klocust/template.go

```diff
@@ -0,0 +1,81 @@
+package klocust
+
+import (
+	"github.com/DevopsArtFactory/klocust/internal/util"
+	"github.com/Masterminds/sprig"
+	"html/template"
+	"os"
+)
+
+type Values struct {
```

Comment:
If this is the template file which a user should make, then I think it is better to split this struct to a different file so that we can manage the values only in one file.

This is helpful when you create a documentation file for this structure.
You can find the example here
https://github.com/DevopsArtFactory/goployer/blob/main/pkg/schemas/config.go

Reply:
Oh, I understood. I've divide LocustValues struct to schemas package.